### PR TITLE
Fix found->find typo

### PR DESCRIPTION
**Pull request by ccaviness**
#### Summary
None

#### Purpose of change
Fix a grammatical error.

#### Describe the solution
Use the correct word.

#### Describe alternatives you've considered
Doing nothing.

#### Testing
None.

### CODE REVIEW

**Review by Maleclypse — APPROVED** [review]

File: data/json/npcs/refugee_center/surface_staff/NPC_old_guard_doctor.json
Comment (actual-nh) [inline data/json/npcs/refugee_center/surface_staff/NPC_old_guard_doctor.json:148]:
```suggestion
      "offer": "So there look to be months, maybe years of experiments, and that data set must be huge.  Database servers massive enough to house it would overheat running on emergency power.  But I did find communications from a lab that had some kind of freezing portal open during the Cataclysm, sending everything to subzero temperatures.  I bet the archives inside that lab are still working.",
```

Comment (ccaviness) [reply data/json/npcs/refugee_center/surface_staff/NPC_old_guard_doctor.json:148]: I believe "looks" is correct here.

Comment (actual-nh) [reply data/json/npcs/refugee_center/surface_staff/NPC_old_guard_doctor.json:148]: It's referring to something plural - "months"/"years". For a second opinion - @ToxiClay?

Comment (ToxiClay) [reply data/json/npcs/refugee_center/surface_staff/NPC_old_guard_doctor.json:148]: The verb "look" here isn't semantically linked to either "months" or "years." For example, you could construct a sentence this way:

`It looks like they're all dead.` Following your logic, you'd say "It look like they're all dead," and yet that clearly looks wrong.

Conversely, `They look like they're all dead.` The semantic link is to the prior word.

Granted, in this example there's a good deal of ambiguity; it's not as clear as in my example sentences where the verb is looking for its link. I don't have a particular rule or convention I'm basing this on, but I would favor the original writing.

```suggestion
      "offer": "So there looks to be months', possibly years' worth of experiments; the resulting data set must be huge.  Any database server massive enough to store it all would have long since overheated if they were left to run on emergency power.  However, I did find communications records from a lab that had some kind of freezing portal open during the Cataclysm, bringing the temperature down below zero.  I bet the archives inside that lab are still working.",
```

**Comment by actual-nh** [comment]
ToxiClay has done a further review, currently buried above as "resolved".

**Comment by Maleclypse** [comment]
> ToxiClay has done a further review, currently buried above as "resolved".

Do you need me to remove my approve then?

**Comment by actual-nh** [comment]
> > ToxiClay has done a further review, currently buried above as "resolved".
> 
> Do you need me to remove my approve then?

Up to you, but probably so; someone with maintainer access also needs to trigger at least the JSON and PR workflows, thanks to Github being annoying...

**Comment by Maleclypse** [comment]
> > > ToxiClay has done a further review, currently buried above as "resolved".
> > 
> > 
> > Do you need me to remove my approve then?
> 
> Up to you, but probably so; someone with maintainer access also needs to trigger at least the JSON and PR workflows, thanks to Github being annoying...

Done and done.

**Pull request merged**
